about.html

```diff
@@ -0,0 +1,243 @@
+        <p class="desktop-about-logo-paragraph">The logo of CC Global Summit 2015 was decided through the logo competition from June 8 to July 7. After consultation with the winning designer, the logo of CC Global Summit 2015, which was optimized for use on the website and promotional materials, is as follows.</p>
+        
+      </div>
+    </div>
+    <div class="row align-items-center mx-5 mb-5 mt-5 ">
+      <div class="col">
+        <img class="logo-about" src="images/about_logo.png" alt="about-logo">
+      </div>
+    </div>
+  </div>
+</section>
+<!-- Secondary About information Section Ends  -->
+
+<!-- Past Speakers Section Starts  -->
+<section class="past-speakers-about">
+  <div class="container pb-5">
+    <div class="row text-center mt-5 pt-5">
+      <div class="col">
+        <p class="past-speakers-title-about">See the past CC Global Summits</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col text-center line pb-5"><i class="fa-solid fa-ellipsis"></i><i class="fa-solid fa-ellipsis"></i><i class="fa-solid fa-ellipsis"></i></div>
+    </div>
+    <div class="row text-center px-5 pb-5">
+      <div class="col">
+        <p>Take a look at the last two CC Summits which took place in buenos Aires and in Warsaw</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <img class="event-2013" src="images/about_2013_hover.png" alt="2013-event">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <img class="event-2011" src="images/about_2011_hover.png" alt="2013-event">
+      </div>
+    </div>
+    <div class="row desktop-event-pics">
+      <div class="col-6 event-1">
+        <img class="desktop-event-2013" src="images/about_2013_hover.png" alt="2013-event">
+      </div>
+      <div class="col-6 event-2">
+        <img class="desktop-event-2011" src="images/about_2011_hover.png" alt="2013-event">
+      </div>
+    </div>
+  </div>
+</section>
+<!-- Past Speakers Section Ends  -->
+
+<!-- Partner Section Starts here -->
+
+<section class="partner-about">
+<div class="container-fluid px-5 pb-5">
+  <div class="row">
+    <div class="col text-white text-center pt-5">Partner</div>
+    </div>
+    <div class="row">
+    <div class="col text-center line pb-5"><i class="fa-solid fa-ellipsis"></i><i class="fa-solid fa-ellipsis"></i><i class="fa-solid fa-ellipsis"></i></div>
+  </div>
+  <div class="row text-center px-2">
+    <div class="col-4">
+      <img src="images/mozilla.PNG" alt="mozilla">
+    </div>
+    <div class="col-4">
+      <img src="images/google.PNG" alt="google">
+    </div>
+    <div class="col-4">
+      <img src="images/naver.PNG" alt="naver">
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-6  text-right pt-2">
+      <img src="images/daum.PNG" alt="daum">
+    </div>
+    <div class="col-6">
+      <img src="images/airbnb.PNG" alt="airbnb">
+    </div>
```

Comment:
- [ ] Kindly your images are overflows the page, kindly control the width of every image to not overflow and break your design.

![image](https://user-images.githubusercontent.com/35535130/157799862-8fd94201-3986-40e6-9040-07f1a9b86f82.png)

Reply:
done
![de](https://user-images.githubusercontent.com/40554384/157904696-befe4124-597b-488b-b3f2-89c49c02a4d4.PNG)